File: CPP/BEE-1036_Formula_de_Bhaskara.cpp
/*--------------------*
| Rodrigo CavanhaMan  |
| BEE 1036            |
| Formula de Bhaskara |
*---------------------*/
#include<stdio.h>
#include<math.h>
using namespace std;

int main() {
	double a, b, c, delta,bhaskara1,bhaskara2;
	scanf("%lf",&a);
	scanf("%lf",&b);
	scanf("%lf",&c);

	delta = ((b*b) - (4*a*c));

	if (delta < 0 || a == 0)
		printf("Impossivel calcular\n");	
	else{
		bhaskara1 = ((b*-1) + sqrt(delta))/(2*a);
		bhaskara2 = ((b*-1) - sqrt(delta))/(2*a);
		printf("R1 = %.5lf\n",bhaskara1);	
		printf("R2 = %.5lf\n",bhaskara2);	
	}
	return 0;
}

File: BEE-1036_Formula_de_Bhaskara.java
/*-------------------*
| Rodrigo CavanhaMan |
|        IFTM        |
|      BEE 1036      |
*--------------------*/
import java.io.IOException;import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args)  throws IOException {
		Scanner sc = new Scanner(System.in);
		sc.useLocale(Locale.ENGLISH);
		Locale.setDefault(new Locale("en", "US"));
		
		double a, b, c, delta,bhaskara1,bhaskara2;
		a = sc.nextDouble();
		b = sc.nextDouble();
		c = sc.nextDouble();

		delta = ((b*b) - (4*a*c));
		
		if (delta < 0 || a == 0){
			System.out.printf("Impossivel calcular\n");	
		}
		else{
			bhaskara1 = ((b*-1) + Math.sqrt(delta))/(2*a);
			bhaskara2 = ((b*-1) - Math.sqrt(delta))/(2*a);

			System.out.printf("R1 = %.5f\n",bhaskara1);	
			System.out.printf("R2 = %.5f\n",bhaskara2);	
		}

		sc.close();
	}
}